rollup-interface/src/state_machine/crypto/simple_hasher.rs

```diff
@@ -0,0 +1,50 @@
+use digest::{generic_array::GenericArray, Digest, OutputSizeUser};
+
+/// A minimal trait representing a hash function. We implement our own
+/// rather than relying on `Digest` for broader compatibility.
+pub trait SimpleHasher: Sized {
```

Comment:
Cool our own trait! Can you please grep our code and replace any `jmt::SimpleHasher` with this `SimpleHasher `

Reply:
Good thinking. Will do!

Reply:
I actually switched to re-exporting `jmt::SimpleHasher` instead of making a custom version. Otherwise, we'll introduce conflicts with the default implementations we want (a `Digest` should be one of our `SimpleHasher`s, but it also has a blanket impl of `jmt::SimpleHasher`).